File: UI/eairflow_desktop/lib/models/mealtype.dart
class MealType {
  int? mealTypeId;
  String? name;
  double? price;

  MealType({this.mealTypeId, this.name, this.price});

  factory MealType.fromJson(Map<String, dynamic> json) {
    return MealType(
      mealTypeId: json["mealTypeId"],
      name: json["name"],
      price: (json["price"] ?? 0).toDouble(),
    );
  }
}

File: eAirFlow.Services/Database/MealType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eAirFlow.Services.Database
{
    public class MealType
    {
        public int MealTypeId { get; set; }
        public string? Name { get; set; }
        public decimal? Price { get; set; }
    }
}
